ansible/roles/debops.prometheus_exporters/defaults/main.yml

```diff
@@ -14,9 +14,9 @@
 #
 # List of base packages to install.
 prometheus_exporters__base_packages:
-  "{{ 'prometheus-node-exporter'     if 'node' in prometheus_exporters__exporters else []
-    + 'prometheus-mysqld-exporter'   if 'mysqld' in prometheus_exporters__exporters else []
-    + [ 'golang', 'libsystemd-dev' ] if 'postfix' in prometheus_exporters__exporters else [] }}"
+  "{{ ([ 'prometheus-node-exporter'   ] if 'node' in prometheus_exporters__exporters else [])
+    + ([ 'prometheus-mysqld-exporter' ] if 'mysqld' in prometheus_exporters__exporters else [])
+    + ([ 'golang', 'libsystemd-dev', 'build-essential' ] if 'postfix' in prometheus_exporters__exporters else []) }}"
```

Comment:
Perhaps relying on `debops.golang` to pull Go dependencies conditionally could be better, the role can setup APT preferences to install Go support from backports. Remember to include its config for `debops.apt_preferences` in the playbook.